src/libraries/System.Net.Http/tests/UnitTests/Headers/AltSvcHeaderParserTest.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net.Http.Headers;
+using Xunit;
+using Xunit.Sdk;
+
+namespace System.Net.Http.Tests
+{
+    public class AltSvcHeaderParserTest
+    {
+        [Theory]
+        [MemberData(nameof(SuccessfulParseData))]
+        public void TryParse_Success(string value, AltSvcHeaderValue[] expectedServices)
```

Comment:
Should have some negative tests here.

Reply:
Agreed, tests as a whole are incomplete and will be filled in over time.